docker-compose.yml

```diff
@@ -20,13 +30,14 @@ services:
       POSTGRES_PASSWORD: netrunnerdb
       RAILS_ENV: $RAILS_ENV
 
-    # makes the app container aware of the DB container
-    links:
+    # starts the app container after the DB and netrunner_cards_json containers
+    depends_on:
       - db
+      - netrunner_cards_json
 
     volumes:
-      - .:/var/www/nrdb-api/
```

Comment:
@plural I suppose you were using this to "hot reload" some of the files while the container is running ?

Reply:
exactly, for dev purposes.  i can use a devcontainer for that instead.